examples/06-verif-manual/vm-007-plastic_compression_of_a_pipe_assembly.py

```diff
@@ -0,0 +1,581 @@
+# Define Young's moulus and Poisson ratio for Steel.
+mapdl.mp("EX", 1, 26.875E6)
+mapdl.mp("PRXY", 1, 0.3)
+
+# Define non-linear material properties for Steel.
+mapdl.tb("BKIN", 1, 1)
+mapdl.tbtemp(0)
+mapdl.tbdata(1, 86000, 0)
+
+# aluminum material model.
+# Define Young's moulus and Poisson ratio for Aluminum.
+mapdl.mp("EX", 2, 11E6)
+mapdl.mp("PRXY", 2, 0.3)
+
+# Define non-linear material properties for Aluminum.
+mapdl.tb("BKIN", 2, 1)
+mapdl.tbtemp(0)
+mapdl.tbdata(1, 55000, 0)
+
+# Print
+print(mapdl.mplist())
+
+
+###############################################################################
+# Plot Stress - Strain Curve
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Use Matplotlib library to represent material model curves of Steel and Aluminum.
+
+# Define stress - strain properties of the steel.
+steel = {"stress_s": [0, 86000, 86000, 86000],
+         "strain_s": [0, 0.032, 0.1, 0.2]}
+
+# Define yielding strength point of the steel on the curve.
+xp = steel["strain_s"][1]
+yp = steel["stress_s"][1]
+
+# Set up the settings of the steel curve.
+plt.plot(steel["strain_s"], steel["stress_s"],
+         label="1020 CR STEEL", linewidth=2,
+         color='steelblue', linestyle="-", marker="o")
+plt.plot(xp, yp, marker='o')
+
+# Annotation settings
+plt.annotate(r'${(\sigma_{yp})_s}$', xy=(xp, yp), xytext=(0.05, 75000),
+             arrowprops=dict(facecolor='steelblue', shrink=0.05),
+             bbox=dict(facecolor='steelblue', edgecolor='black', boxstyle='round, pad=1'))
+
+# Define stress - strain properties of the aluminum.
+aluminum = {"stress_a": [0, 55000, 55000, 55000],
+            "strain_a": [0, 0.05, 0.1, 0.2]}
+
+# Define yielding strength point of the Aluminum on the curve.
+xp = aluminum["strain_a"][1]
+yp = aluminum["stress_a"][1]
+
+# Set up the settings of the aluminum curve.
+plt.plot(aluminum["strain_a"], aluminum["stress_a"],
+         label="2024-T4 Aluminum", linewidth=2,
+         color='sandybrown', linestyle="-", marker="o")
+plt.plot(xp, yp, marker='o')
+
+# Annotation settings
+plt.annotate(r'${(\sigma_{yp})_a}$', xy=(xp, yp), xytext=(0.07, 45000),
+             arrowprops=dict(facecolor='sandybrown', shrink=0.05),
+             bbox=dict(facecolor='sandybrown', edgecolor='black', boxstyle='round, pad=1'))
+
+plt.grid(True)
+plt.legend()
+plt.title("Stress - Strain Curve", fontsize=18)
+plt.show()
+
+
+###############################################################################
+# Define Section
+# ~~~~~~~~~~~~~~
+# Set up the cross-section properties for a shell and pipe elements.
+
+# Shell cross-section for inside (STEEL) tube.
+mapdl.sectype(1, "SHELL")
+
+# Thickness (SHELL181)
+mapdl.secdata(0.5, 1, 0, 5)
+
+# Shell cross-section for outside (Aluminum) tube.
+mapdl.sectype(2, "SHELL")
+
+# Thickness (SHELL181)
+mapdl.secdata(0.5, 2, 0, 5)
+
+# Define Pipe cross-section for inside (STEEL) tube, where:
+mapdl.sectype(3, "PIPE")
+
+# Outside diameter and wall thickness settings for Inside Tube (PIPE288)
```

Comment:
You should be consistent with capitalization. Only names or **very** important words should be in capital.
```suggestion
# Outside diameter and wall thickness settings for inside Tube (PIPE288)
```